transport-classes-io_uring/src/main/java/io/netty/incubator/channel/uring/IOUringEventLoop.java

```diff
@@ -251,10 +251,10 @@ private void handle(int fd, int res, int flags, byte op, short data) {
             if (channel == null) {
                 return;
             }
-            if (op == Native.IORING_OP_READ || op == Native.IORING_OP_ACCEPT || op == Native.IORING_OP_RECVMSG) {
+            if (op == Native.IORING_OP_RECV || op == Native.IORING_OP_ACCEPT || op == Native.IORING_OP_RECVMSG) {
                 handleRead(channel, res, data);
             } else if (op == Native.IORING_OP_WRITEV ||
-                    op == Native.IORING_OP_WRITE || op == Native.IORING_OP_SENDMSG) {
+                    op == Native.IORING_OP_SEND || op == Native.IORING_OP_SENDMSG) {
```

Comment:
Maybe just leave the write here as well and the read below. Just in case we reuse these again in the future

Reply:
Good point

Reply:
@franz1981 let me know once done

Reply:
done!